### ISSUE TITLE
css table cell padding rendering

### ISSUE THREAD

**Issue by manuelmeister** [web-platform-tests/wpt#32324]
Safari has a bug that renders the padding of a cell incorrectly: https://bugs.webkit.org/show_bug.cgi?id=78905

![Green square with red rectangles on top and bottom](https://user-images.githubusercontent.com/3001985/148944691-921b553d-865c-47ec-b1bd-2a311652e8e8.png)
(Test case by tabatkins from the bug report).

I searched for such a test, but couldn't find one. Should I add a test case? If so, where? under css2 or css-tables?

**Comment by tabatkins** [comment]
Yeah that issue predates WPT, so it's very possible there's no test for it. Feel free to add one, I suppose under css2? @fantasai or @svgeesus, do y'all have an opinion?

**Comment by tabatkins** [comment]
Ah indeed, and that's a vital part, so css-tables then
